server/cluster/coordinator.go

```diff
@@ -155,6 +156,35 @@ func (c *coordinator) patrolRegions() {
 	}
 }
 
+// checkPriorityRegions
```

Comment:
please complete comment or simply remove it.

Reply:
I will complete it